libandroid-navigation/src/test/java/com/mapbox/services/android/navigation/v5/internal/navigation/metrics/SchemaTest.java

```diff
@@ -90,6 +91,7 @@ public void checkNavigationDepartEventFields() throws Exception {
   }
 
   @Test
+  @Ignore
```

Comment:
Could you add a `TODO` so we don't forget to address as part of https://github.com/mapbox/mapbox-navigation-android/issues/2923